File: src/main/java/com/learnershi/rclasssocket/repository/UserClassRoomsRepository.java
package com.learnershi.rclasssocket.repository;


import com.learnershi.rclasssocket.entity.UserClassRooms;
import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
import reactor.core.publisher.Mono;

public interface UserClassRoomsRepository extends ReactiveMongoRepository<UserClassRooms, String> {
    Mono<UserClassRooms> findByUserSeq(String userSeq);
}

File: src/main/java/com/learnershi/rclasssocket/repository/UserClassRoomsRepository.kt
package com.learnershi.rclasssocket.repository

import com.learnershi.rclasssocket.entity.UserClassRooms
import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import reactor.core.publisher.Mono

interface UserClassRoomsRepository : ReactiveMongoRepository<UserClassRooms?, String?> {
    fun findByUserSeq(userSeq: String?): Mono<UserClassRooms?>
}